railties/lib/rails/commands/credentials/credentials_command.rb

```diff
@@ -110,18 +110,19 @@ def warn_if_credentials_are_invalid
 
         def missing_credentials_message
           if !credentials.key?
-            "Missing '#{key_path}' to decrypt credentials. See `#{executable(:help)}`"
+            friendly_key_path = key_path.sub(/^#{Rails.application.root}\//, "")
```

Comment:
Without this, the path is a full path (from [here](https://github.com/rails/rails/blob/ec782106553202e7cf2aeff7cbbce94d85b75635/railties/lib/rails/application/configuration.rb#L557)) which breaks [this test](https://github.com/rails/rails/blob/ec782106553202e7cf2aeff7cbbce94d85b75635/railties/test/commands/credentials_test.rb#L178).

Reply:
There's probably a neater way to write it...

Reply:
Perhaps `friendly_key_path = key_path.delete_prefix(Rails.application.root)` is a bit more elegant, because it won't break if there happens to be a regex character in the path.

Reply:
Maybe could use `Pathname#relative_path_from`?